README.md

```diff
@@ -1,11 +1,15 @@
 # Kottans-frontend
+
```

Comment:
It's probably even the right changes by some of the configs, but too many of them - 304 lines added 179 lines deleted?